pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/BacklogQuota.java

```diff
@@ -28,16 +28,27 @@
  * <code>limit</code> representing a quota limit in bytes and <code>policy</code> for backlog retention policy.
  */
 public class BacklogQuota {
-    private long limit;
+    private long limitSize;
+    // backlog quota by time in second
+    private int limitTime;
     private RetentionPolicy policy;
 
     /**
-     * Gets quota limit in bytes.
+     * Gets quota limit in size.
      *
      * @return quota limit in bytes
      */
-    public long getLimit() {
-        return limit;
+    public long getLimitSize() {
```

Comment:
This object is serialized, changing the name is a breaking change.
Are we handling compatibility with old configurations?

Reply:
Good catch, I actually missed that. Added compatibility conversion and a unit test to make sure it works.